documentation/book/api/index.adoc

```diff
@@ -1386,6 +1386,7 @@ Retrieves the bridge metrics in Prometheus format.
 ===== Content Type
 
 * text/plain
+* application/json
```

Comment:
Do we support JSON based metrics now? How?

Reply:
See my reply to your other comment. When JSON will be removed from the openapi spec as well, it will be removed from here.